src/raygun.rum/index.js

```diff
@@ -83,6 +83,8 @@ var raygunRumFactory = function(window, $, Raygun) {
       CustomTiming: 't'
     };
 
+    this.Utilities = {};
```

Comment:
I quite like this way of storing utility methods in a utilities object! 👍

Reply:
Ideally they would probably be stored in a seperate file and then imported/used as required, since they are largely independent of this file but this requires a more major overhaul of how we compile/bundle this library.

This Utilities is a work around at this stage to get access to the more "private" methods hidden in the closure.